samples/name_domains_values/Coded_Domain_Cyberspace.csv

```diff
@@ -1,2 +0,0 @@
-Name,Value
```

Comment:
Its starting to look like many domains we actually use got inadvertently removed in this PR: 

Compare to https://github.com/Esri/joint-military-symbology-xml/tree/esri-implementation/samples/name_domains_values